### PR TITLE
281 did you notice

### PR DESCRIPTION
**Pull request by heathermoses**
- [x] No discrepancies across browsers. Eg. chrome vs safari
- [x] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [x] CSS (https://jigsaw.w3.org/css-validator/)
- [x] Google sign-in is operational
- [x] Accessibility functions
- [x] Pages can scale without distorting page
- [x] No dead links
- [x] Navbar is consistent across the site
- [x] Pages are screen-reader accessible
- [x] Contrast meets standards for accessibility
- [x] Pages are keyboard accessible
- [x] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: client/src/components/exercise/lab8/pages/BiasDiscovery.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          data that contained both appropriate and inappropriate messages, so
+          why did the AI make these mistakes?
+        </div>
+        <div className="playthrough__sentence">
+          When the data used to train AI is not equitable, selection bias can
+          occur. Selection bias is a result of datasets that are “not reflective
+          of their real-world distribution” and are not accurately
+          representative. (
+          <a
+            href="https://developers.google.com/machine-learning/crash-course/fairness/types-of-bias"
```
Comment (2nPlusOne) [inline client/src/components/exercise/lab8/pages/BiasDiscovery.js:33]: Great reinforcement!

File: client/src/components/exercise/lab8/pages/BiasDiscovery.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            Fairness: Types of Bias
+          </a>
+          )
+        </div>
+        <div className="playthrough__sentence">
+          In this situation, the AI was trained on a general dataset that did
+          not contain data specific to MineALL, which is not accurately
+          representative. Due to the lack of data specific to MineALL, the AI
+          recommended messages that are inappropriate in a general context to be
+          removed, but are considered appropriate in the context of MineALL.
```
Comment (2nPlusOne) [inline client/src/components/exercise/lab8/pages/BiasDiscovery.js]: How do you feel about this change to the last sentence: "... to be removed, even though they are considered appropriate in the context of MineALL." ?

**Review by 2nPlusOne — APPROVED** [review]
Looking good! I only had one small discussion point regarding the last paragraph.

File: client/src/components/exercise/lab8/pages/BiasDiscovery.js (hunk 2 above)
Comment (heathermoses) [reply client/src/components/exercise/lab8/pages/BiasDiscovery.js]: That's a great idea, good point!

**Review by Stormkiller-01 — APPROVED** [review]

**Pull request merged**